sixtyfps_runtime/corelib/graphics/brush.rs

```diff
@@ -15,6 +15,9 @@ use super::{Color, Point};
 use crate::properties::InterpolatedPropertyValue;
 use crate::SharedVector;
 
+#[cfg(not(feature = "std"))]
```

Comment:
This could also be `feature = "libm"` instead, but then... it's no_std that disables the float point ops it seems. I'm not sure how this should work identically on SOCs that do have a hardware floating point unit and us combining that with no_std. Maybe the ops *are* available then?